### DISCUSSION TITLE
Cut and extract part of the osgb model, enlarge the result with the mouse wheel, some tiles will suddenly disappear

### DISCUSSION THREAD

**Discussion by letmesignup** [openscenegraph/OpenSceneGraph#1226] [Q&A]
Hello! 
I'm a beginner for OSG, and I met a problem that is when I cut and extract part of the osgb model, then enlarge the result with the mouse wheel, some tiles will suddenly disappear. I discussed with some people yesterday, and they said maybe the osgb Center and Bounding Box is wrong, but I don't know how to fix it specifically. Could someone tell me something about this?
Thank you!

this is the problem gif link. Copy it and open it in your browser, you will see.
[https://media.giphy.com/media/20Su5vxY1RFXhIaMg7/giphy.gif](url)

**Comment by robertosfield** [comment]
There is no way anyone can help you with the information provided, it's far too vague.

The OSG doesn't have a "cut and extract part of model feature" so I can only presume you or someone else has written this. This means there is no way anyone else can know what has been done in this code ad what it may have done wrong.  The link is also just a 404.

**Reply by letmesignup** [reply]
Thank you for replying, indeed my question is unclear, I will do more research about OSG and this problem, but the question is as I said I'm New for OSG, so I don't know where to start to fix this problem.  I can visit this link if  I Ctrl+C the link and Ctrl+V on the edge's Adress bar, and I don't know the other way to show a picture or video for you.  Do you have any good suggestion for uploading GIF, picture or video? 
Thank you again.
